build.sbt

```diff
@@ -31,23 +28,22 @@ lazy val microservice = (project in file("."))
     IntegrationTest / parallelExecution := false
   )
   .settings(scoverageSettings)
-  .disablePlugins(JUnitXmlReportPlugin) //Required to prevent https://github.com/scalatest/scalatest/issues/1427
+  .disablePlugins(JUnitXmlReportPlugin) // Required to prevent https://github.com/scalatest/scalatest/issues/1427
 
-lazy val commonSettings = Seq(
-  scalaVersion := "2.13.8",
-  scalacOptions ++= scalacFlags,
-  libraryDependencies ++= Dependencies.compile ++ Dependencies.test
-)
+lazy val commonSettings =
+  Seq(scalaVersion := "2.13.12", scalacOptions ++= scalacFlags, libraryDependencies ++= Dependencies.compile ++ Dependencies.test)
```

Comment:
Do not know if this change is due to some auto reformatting, but honestly it was more readable as it was before

Reply:
The only thing I changed in this file is the scalaVersion number and the silencing of the "private val ..." in scalaFlags, the rest is scalafmt being wacky😞